documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md

```diff
@@ -99,11 +99,11 @@ Parameter | Type  | Description
 
 You can also specify additional parameters for the action. See [Drag To Element Action Options](action-options.md#drag-to-element-action-options).
 
-> You can add this action to the test only in the Test Editor, you cannot record it.
+> You can add this action to the test only in the Test Editor.
```

Comment:
@dirk-pieterse 

I think we should use my phrase. Otherwise, the phrase is not full. What means "only in the Test Editor"? The user can think about other editors. However, we mean that "you cannot record the action using the visual recorder".

Reply:
You can use the Test Editor to add this action to the test- you cannot record this action.